app/code/Magento/Paypal/Model/Payflow/Transparent.php

```diff
@@ -203,9 +203,9 @@ public function authorize(InfoInterface $payment, $amount)
         $payPalCart = $this->payPalCartFactory->create(['salesModel' => $order]);
         $payPalCart->getAmounts();
 
-        $parentTransactionId = $vaultPaymentToken ? $vaultPaymentToken->getGatewayToken() : $zeroAmountAuthorizationId;
+        $token = $payment->getAdditionalInformation(self::PNREF);
```

Comment:
This change will break a case when the order is in Suspected Fraud status and Admin user accepts payment.
\Magento\Paypal\Model\Payflow\Transparent::acceptPayment method calls \Magento\Paypal\Model\Payflow\Transparent::capture and there is no $payment->getAdditionalInformation(self::PNREF) value on that stage. So we need to take reference transaction id from stored token.

Reply:
@viktym Makes sense. Updating PR to remove this change.